lambda-function.js

```diff
@@ -0,0 +1,35 @@
+module.exports = lambdaFunction;
+
+async function lambdaFunction(probot, event, context) {
+  try {
+    // Ends function immediately after callback
+    context.callbackWaitsForEmptyEventLoop = false;
+
+    // this could will be simpler once we  ship `verifyAndParse()`
+    // see https://github.com/octokit/webhooks.js/issues/379
+    await probot.webhooks.verifyAndReceive({
+      id:
+        event.headers["X-GitHub-Delivery"] ||
+        event.headers["x-github-delivery"],
+      name: event.headers["X-GitHub-Event"] || event.headers["x-github-event"],
+      signature:
+        event.headers["X-Hub-Signature-256"] ||
+        event.headers["x-hub-signature-256"] ||
+        event.headers["X-Hub-Signature"] ||
+        event.headers["x-hub-signature"],
+      payload: JSON.parse(event.body),
+    });
+
+    return {
+      statusCode: 200,
+      body: '{"ok":true}',
+    };
+  } catch (error) {
+    probot.log.error(error);
+
```

Comment:
Actually we already log an error if `verifyAndReceive` fails, so this will result in a double log

```suggestion
```